packages/terra-graphs/package.json

```diff
@@ -18,22 +15,114 @@
     "terra-graphs",
     "carbon",
     "graphs",
-    "UI"
+    "UI",
+    "d3",
+    "d3-module",
+    "graphing"
   ],
   "author": "Cerner Corporation",
   "license": "Apache-2.0",
   "bugs": {
     "url": "https://github.com/cerner/terra-graphs/issues"
   },
   "homepage": "https://github.com/cerner/terra-graphs#readme",
-  "peerDependencies": {
-    "react": "^16.8.5",
-    "react-dom": "^16.8.5"
+  "files": [
+    "dist",
+    "lib",
+    "README.md"
+  ],
+  "prettier": {
+    "arrowParens": "always",
+    "bracketSpacing": true,
+    "semi": true,
+    "tabWidth": 4,
+    "trailingComma": "none",
+    "overrides": [
+      {
+        "files": "package.json",
+        "options": {
+          "tabWidth": 2,
+          "printWidth": 250
+        }
+      }
+    ]
+  },
+  "publishConfig": {
+    "access": "public"
   },
   "dependencies": {
     "classnames": "^2.2.5",
     "prop-types": "^15.5.8",
-    "terra-theme-context": "^1.0.0"
+    "terra-theme-context": "^1.0.0",
+    "d3": "^5.16.0"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.10.4",
+    "@babel/plugin-transform-classes": "^7.10.4",
+    "@babel/plugin-transform-runtime": "^7.10.4",
+    "@babel/polyfill": "^7.10.4",
+    "@babel/preset-env": "^7.10.4",
+    "@babel/runtime": "^7.10.4",
+    "@cerner/carbon-site-helpers": "^1.11.3",
+    "@cerner/svg-to-carbon": "^1.1.3",
+    "@commitlint/cli": "^8.3.5",
+    "@commitlint/config-conventional": "^8.3.4",
+    "@semantic-release/git": "^9.0.0",
+    "autoprefixer": "^9.8.4",
+    "babel-loader": "^8.1.0",
+    "babel-plugin-istanbul": "^6.0.0",
+    "babel-plugin-minify-replace": "^0.5.0",
+    "babel-preset-minify": "^0.5.1",
+    "babel-eslint": "^10.1.0",
+    "commander": "^5.1.0",
+    "conventional-changelog-eslint": "^3.0.8",
+    "css-loader": "^3.6.0",
+    "cssnano": "^4.1.7",
+    "docdash": "^1.2.0",
+    "eslint": "^6.8.0",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-config-standard": "^14.1.1",
+    "eslint-plugin-import": "^2.22.0",
+    "eslint-plugin-jsdoc": "^24.0.6",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^4.2.1",
+    "eslint-plugin-standard": "^4.0.1",
```

Comment:
These should have been defined at the root level already and can be removed here. Bootstrap will pull these up for package-level testing

Reply:
Yup, they have been moved up.